src/static/riot/competitions/editor/form.tag

```diff
@@ -30,7 +30,11 @@
                     </div>
                     <errors errors="{errors}"></errors>
                 </div>
-
+                
+                <button class="ui primary button">
+                    <a style="color: #fff" href="{URLS.COMPETITION_DETAIL(opts.competition_id)}">Back To Competition</a>
```

Comment:
Let's avoid inline styling

Reply:
Don't think we need this button at all, since it is a duplicate of the one below, in the group of all the other buttons.